repos/system_upgrade/el7toel8/actors/grubbydebug/actor.py

```diff
@@ -45,3 +45,19 @@ def enable_debug(filename):
                 except (IOError, OSError):
                     api.current_logger().warning(
                         'Failed to activate debug mode in {}'.format(file_), exc_info=True)
+
+        try:
+            run(['ls', '-la', '/boot'])
+        except CalledProcessError:
+            api.current_logger().warning(
+                'Could not list /boot dir', exc_info=True)
+        try:
+            run(['cat', '/boot/loader/entries/*.conf'])
```

Comment:
The asterisk won't expand (...aand we got `Traceback` in tests.).

```suggestion
            run(['sh', '-c', 'cat /boot/loader/entries/*.conf'])
```

Reply:
Yep, forgot about this :) 

Long live the PR e2e tests \o/

Reply:
Anyway, it is nice that the word "Traceback" is nicely formatted with back-quotes but I would more appreciate seeing the traceback itself :) So I can cover it properly on top of the fixing the command itself...